lang/lfortran/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup               cmake 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               github 1.0
 
-github.setup            lfortran lfortran 0.23.0 v
```

Comment:
latest version is 0.28.0

Reply:
also, why can you not use the normal way of downloading the sources. `fetch.type git` is really not something that should be used... The description of why this is no needed is not very clear - did you try to convince upstream to fix their build system?

Reply:
@reneeotten Oh wow, they are fast. It was at 0.25.0 yesterday.

Let me check if normal way works now. Maybe the issue has been resolved, w/e it was.

Reply:
Switched to tarball.